dojo-starter/src/systems/actions.cairo

```diff
@@ -19,9 +20,12 @@ mod actions {
             let player = get_caller_address();
             // create a new object of Game and get the unique Id of the Game and return
             let game_id = world.uuid();
-            let new_game : Game = Game{id: game_id, player: player, score: 0}; 
+            let new_game : Game = Game{id: game_id, player: player, score: 0, state: GameStage::InProgress}; 
             return new_game;
         }
+        fn end_game(game: Game) {
+            game::state = GameStage::Over;
```

Comment:
here you get the id, then get the game using that id, then update the game, and set it again in the world

Reply:
the problem is how do we get the game object using the Id. need to maintain some in-memory colllection.

Reply:
https://github.com/Mugen-Space/dojo-game-jam-v1/blob/main/dojo-starter/src/systems/actions.cairo#L123

example regarding how we can get it using the ID and also the next fews line shows how can we update the state of the given game